### DISCUSSION TITLE
Why framework Serverless are creating multiple deployment buckets

### DISCUSSION THREAD

**Discussion by robsonalvesbh** [serverless/serverless#9837] [Q&A]
I noticed that some deployments end up creating a new bucket and over time s3 gets full of buckets of deploys from the same lambda.

Could someone explain why this happens? I'm making something wrong?

![image](https://user-images.githubusercontent.com/9202301/129246046-4a1230ce-087f-4ed4-b1f1-7420721b2638.png)

**Comment by pgrzesik** [comment]
Could you share a bit more on how you're deploying your services? It looks like instead of deploying things as a part of a single service you create new ones and they will use different buckets.
